controllers/monitoring.go

```diff
@@ -106,8 +106,7 @@ func (r *ClusterGroupUpgradeReconciler) getMonitoredObjects(managedPolicy *unstr
 			// Filter only Subscription templates.
 			if !isMonitoredObjectType(kind) {
 				r.Log.Info(
-					"[getPolicyContent] Policy spec.policy-templates.objectDefinition.spec.object-templates.kind does not need to be monitored",
-					"policyName", managedPolicyName)
+					"[getPolicyContent] Object does not need to be monitored", "policyName", managedPolicyName, "kind", kind)
```

Comment:
Is this related to the PR? But regardless, can we be a bit more explicit as why it doesn't need to be monitored? `Object does not need to be monitored because...`

Reply:
Right, this is not related. Just noticed that it wasn't even a readable sentence. Right now we only monitor subscriptions. We may start monitoring other kinds in the future. That's why I added the kind to the log. I am actually ok with removing the log altogether if you find it confusing. It was there before I joined this project. I think it has served its purposes during the early days of subscription/install plan related development.

Reply:
Removal sounds good with me.